### PR TITLE
Added Pitch Count

### PR DESCRIPTION
**Pull request by tracerrx**
Added pitch count to the pitcher scroll.  Set show_pitch_count to true/false in config.json to enable/disable

### CODE REVIEW

**Comment by ty-porter** [comment]
Agreed, `Game` objects are data containers. Turning config on or off should swap display options, not what the underlying data includes (unless there's a good reason for that)

**Comment by WardBrian** [comment]
Storing it alongside balls/strikes/pitch speed also allows easier control over where the user would like it to be. For example, I think I'd want it as a separate text element rather than part of the pitcher's name

**Comment by tracerrx** [comment]
I think this moves it to the right place

**Comment by WardBrian** [comment]
How do you feel about this setting being in the layout rather than the config?

**Comment by ty-porter** [comment]
Correct me if I'm wrong... The reason it's a config setting is because we want it to scroll with the existing pitcher text?

So our options are:
* layout (static text)

**Comment by WardBrian** [comment]
I wouldn’t want it on the pitcher text I think, I would want it like the speeds (probably under them, on my layout), but it could be an “off” “pitcher name” “separate” setting in the layout

File: renderers/games/game.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 except ImportError:
     from RGBMatrixEmulator import graphics
 
+from distutils import config
```
Comment (ty-porter) [inline renderers/games/game.py:6]: This import is not needed, probably breaks some stuff

**Comment by tracerrx** [comment]
I understand the logic of having it an independent and being able to move it around the layout... But wouldn't that prohibit it from being part of the pitcher scroll?

**Comment by WardBrian** [comment]
I don’t think so, you should be able to append it right before rendering. 

I can sketch this out in my own fork if it’s helpful, you already did the hard part of finding where the data lives in the API

**Comment by tracerrx** [comment]
So in layout file, under atbat something like `"pitch_count": {
      "font_name": "4x6",
      "x": 1,
      "y": 50,
      "enabled": true,
      "append_pitcher_Name": true
    },`

And if append pitch name==true, don't render separately but append to the pitcher name.

**Comment by WardBrian** [comment]
That sounds good to me

**Comment by tracerrx** [comment]
Sure.. I prob can’t get around to it until Tuesday though

**Comment by WardBrian** [comment]
#431 implemented this feature in a way that only uses the coordinates file, not the config. You can see the example here: 
https://github.com/MLB-LED-Scoreboard/mlb-led-scoreboard/blob/0fee496dfa68232ab07daba329f899f70aeada50/coordinates/w64h32.json.example#L135-L140

**Pull request closed without merging**